README.md

```diff
@@ -1,5 +1,7 @@
 # Homebase React
 
+![Homebase React Logo](public/images/logo-blk.png)
```

Comment:
I don't see this image in the preview

Reply:
yikes thanks for pointing that out. committed it and pushed now